.travis.yml

```diff
@@ -9,23 +9,34 @@ script:
   - cargo build --examples --all
 stages:
```

Comment:
Jobs in the same stage can run in parallel. The old setup is more reasonable, using just two stages, Check and Test.